### ISSUE TITLE
Add Canadian NI Reports Layer

### ISSUE THREAD

**Issue by marshallrobinson** [UGS-GIO/Minerals-Metals#10]
More details to come...

**Comment by StephMills** [comment]
@marshallrobinson I don't think I know where the pdfs for the current map are stored, do you? do you need any info from me in order to move the info from the current map into the Min Resc map? The only feature I would ask is to make sure people can download the pdfs. Let me know what else you need

**Comment by marshallrobinson** [comment]
Yep! We've got access to them on our end. Unless you have extra data/pdfs to hand off to us, then all we need to do on my end is add the data and create some explanatory text to the the intro "About" panels. Is the text from https://geology.utah.gov/apps/reportviewer/index.html okay to pull over (as-is) or do you think it needs to be changed a bit?

**Comment by arupke** [comment]
I think we should probably refresh the explanatory text for the reports... It should probably be broadened at some level.

**Comment by arupke** [comment]
I'll try to get to that in the next few days...

**Comment by arupke** [comment]
Not grant funded...

**Comment by marthajensen** [comment]
@jjhi11 This layer is on AGOL at https://services.arcgis.com/ZzrwjTRez6FJiOq4/arcgis/rest/services/CanadianFeb2020/FeatureServer
It's a layer that Mackenzie created for the Technical Reports viewer (https://geology.utah.gov/apps/reportviewer/index.html)
